src/diffusers/pipelines/pag/__init__.py

```diff
@@ -55,7 +57,6 @@
         from .pipeline_pag_sd import StableDiffusionPAGPipeline
         from .pipeline_pag_sd_3 import StableDiffusion3PAGPipeline
         from .pipeline_pag_sd_animatediff import AnimateDiffPAGPipeline
-        from .pipeline_pag_sd_img2img import StableDiffusionPAGImg2ImgPipeline
```

Comment:
also need to add pag sd inpaint here too -currently CI fail for some import error

Reply:
done